### PR TITLE
[03] Add ISS green pilot time series data getter

### LINKED ISSUE #3: Add time series getter and initial processing

### PR DESCRIPTION
**Pull request by Jack-0-0**
Closes #3.

This PR adds:
- Data getter to load ISS green pilot time series data (`iss_forecasting/getters/iss_green_pilot_ts.py`)
- Processing function to find items in a dataframe with 0s (`utils/processing.py`)
- Notebook file (`analysis/tech_category_level/research_funding_and_investment.py`) which uses above functions and provides some info about the data and the tech categories. This notebook will be used for further analysis.

### CODE REVIEW

**Comment by beingkk** [comment]
Thank you @Jack-0-0, will aim to check it by the end of today!

File: iss_forecasting/analysis/tech_category_level/research_funding_and_investment.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+# %%
+# view sample of dataframe
+iss_ts.tail(5)
+
+# %% [markdown]
+# Column descriptions:<br>
+# `year` = year (between 2007 and 2021)<br>
+# `research_no_of_projects` = number of research projects started in a specific `year` (GtR)<br>
+# `research_funding_total` = research funding amount (GBP 1000s) awarded in a specific `year` (GtR)<br>
+# `investment_rounds` = number of rounds of invesmtnet in a specific `year` (Crunchbase)<br>
```
Comment (beingkk) [inline iss_forecasting/analysis/tech_category_level/research_funding_and_investment.py]: Typo
```suggestion
# `investment_rounds` = number of rounds of investment in a specific `year` (Crunchbase)<br>
```

File: iss_forecasting/analysis/tech_category_level/research_funding_and_investment.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+# `investment_raised_total` = raised investment amount (GBP 1000s) a specific `year` (Crunchbase)<br>
```
Comment (beingkk) [inline iss_forecasting/analysis/tech_category_level/research_funding_and_investment.py]: Typo
```suggestion
# `investment_raised_total` = raised investment amount (GBP 1000s) in a specific `year` (Crunchbase)<br>
```

**Review by beingkk — APPROVED** [review]
Hi, thanks for this! I managed to successfully clone the repo and run `make install` 

Perhaps useful to mention in the readme that one also needs to do `pip install -e .` (or perhaps you have another approach for this...) so that the repo is also recognised as a package. Otherwise lines like `from iss_forecasting.utils.processing import find_zero_items` throw an error for me 

Otherwise looks good (just spotted a couple typos)

**Comment by Jack-0-0** [comment]
> Hi, thanks for this! I managed to successfully clone the repo and run `make install`
> 
> Perhaps useful to mention in the readme that one also needs to do `pip install -e .` (or perhaps you have another approach for this...) so that the repo is also recognised as a package. Otherwise lines like `from iss_forecasting.utils.processing import find_zero_items` throw an error for me
> 
> Otherwise looks good (just spotted a couple typos)

Thanks for the review. I've fixed the typos 🙂

When running `make install` it should run `pip install -e .` (see lines 44-47 and 118-125 in the `Makefile`). It's strange that it didn't do that when you ran `make install`. Have you had this issue before when setting up other cookiecutters?

**Pull request merged**